### ISSUE TITLE
Object tags

### ISSUE THREAD

**Issue by FreHu** [abapGit/abapGit#5360]
Separated from https://github.com/abapGit/abapGit/issues/5355#issuecomment-1042051608

**Comment by mbtools** [comment]
i like the idea. i would use `tests` instead of `ut`. other candidates:
- `ddic`
- `json`
- `js`

nice to have: 
- clicking on tag filters the list

**Comment by sbcgua** [comment]
recap of some initial thoughts:
- imho listing file extensions would be simple and sufficient
- with some extra mappings like `ut` (or `tests` - good idea)
- `ddic` - I also though about it initially. However ddic is a category of the object and not of it's contents. Thus controversial.
- filters - yes, good idea, I'd redefine it a bit - the tags should be searchable by free entry filter field (which is not implemented so far :). clicking them may add the term to the filter. To unify the approach.
- color coding - could be, but imho very controversial and needs experiments. more than 3-4 colors on a page become imperceivable. Also what is the subject for color coding ? Status of files ? Is there much use ? Imho - no, it's minimal added value to the object status. Maybe I'd separate tests to show that object is unit tested. But again what is the added value to the label itself. (to https://github.com/abapGit/abapGit/issues/5355#issuecomment-1042181166 @FreHu ). This subject needs some deeper exploring

**Comment by mbtools** [comment]
Agree. `ddic` is a different categorization. It actually fits the `icon` column, which could also be available for filtering.

I'm undecided on colors for tags. It kind of works when you look at the tags used on GitHub although (or because?) colors are "random" and not associated with "red/yellow/green" semantics.

**Comment by sbcgua** [comment]
in my understanding: github base interface is very neutral and greyish, if you looks at AG repo there are few tags used so in fact we come down to the mentioned 3-4 colors. Actually if you come across a repo with multiple tags assigned to each issue the start to be overwhelming too.
